tensorflow/core/grappler/optimizers/constant_folding.h

```diff
@@ -35,8 +35,10 @@ const char kConstantFoldingCtrl[] = "ConstantFoldingCtrl";
 // Constant folding optimization for a graph.
 class ConstantFolding : public GraphOptimizer {
  public:
+  // The size limit will only be considered if the newly created node is greater
+  // than input_size (optional).
```

Comment:
nit: rename input_size to original_size or something, so it's more clear what this means outside of the context of how you call it.